src/prefect/deployments/runner.py

```diff
@@ -549,7 +549,9 @@ def from_flow(
                         module = importlib.import_module(mod_name)
                         flow_file = getattr(module, "__file__", None)
                     except ModuleNotFoundError as exc:
-                        if "__prefect_loader__" in str(exc):
+                        # 16458 adds an identifier to the module name, so checking
```

Comment:
16458 is _this_ PR :grin: Unless I'm missing something I think this is referencing a different PR?